src/device/rdk/system/settings/get.rs

```diff
@@ -28,31 +27,19 @@ fn get_rdk_language() -> Result<String, DabError> {
 }
 
 fn get_rdk_resolution() -> Result<OutputResolution, DabError> {
-    if get_service_state("org.rdk.FrameRate")? != "activated" {
-        service_activate("org.rdk.FrameRate".to_string())?;
-        thread::sleep(std::time::Duration::from_millis(500));
-    }
+    let displayinfo_width = get_thunder_property("DisplayInfo.width", "")?;
+    let width = displayinfo_width.parse::<u32>().map_err(|_| DabError::Err400("Invalid width(parse to u32 failed)".to_string()))?;
 
-    #[allow(dead_code)]
-    #[derive(Deserialize)]
-    struct GetDisplayFrameRate {
-        framerate: String,
-        success: bool,
-    }
-
-    let rdkresponse: RdkResponse<GetDisplayFrameRate> =
-        rdk_request("org.rdk.FrameRate.getDisplayFrameRate")?;
+    let displayinfo_height = get_thunder_property("DisplayInfo.height", "")?;
+    let height = displayinfo_height.parse::<u32>().map_err(|_| DabError::Err400("Invalid height(parse to u32 failed)".to_string()))?;
 
-    let mut dimensions = rdkresponse
-        .result
-        .framerate
-        .trim_end_matches(']')
-        .split('x');
+    let displayinfo_framerate = get_thunder_property("DisplayInfo.framerate", "")?;
+    let frequency = get_frequency_from_displayinfo_framerate(&displayinfo_framerate).unwrap_or(60.0);
```

Comment:
Parsing of frequency currently doesn't work correctly due to an issue in `get_thunder_property()`. We have to first merge [this fix](https://github.com/device-automation-bus/dab-adapter-rs/pull/58).